<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settingsFormMusicLib</class>
 <widget class="QWidget" name="settingsFormMusicLib">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>145</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>501</width>
     <height>80</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>0</y>
       <width>470</width>
       <height>81</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>161</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Папка для сохранения:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="musiclibPathEdit">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>30</y>
        <width>231</width>
        <height>22</height>
       </rect>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="focusPolicy">
       <enum>Qt::TabFocus</enum>
      </property>
      <property name="acceptDrops">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QToolButton" name="musiclibPathButton">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>30</y>
        <width>27</width>
        <height>22</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::TabFocus</enum>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>0</y>
       <width>470</width>
       <height>80</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>30</y>
        <width>131</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Путь к папке iTunes:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="iTunesMediaPathEdit">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>30</y>
        <width>241</width>
        <height>22</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::TabFocus</enum>
      </property>
     </widget>
     <widget class="QToolButton" name="iTunesMediaPathButton">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>30</y>
        <width>27</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>91</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Сохранять в:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>271</width>
     <height>26</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <item>
    <property name="text">
     <string>Папку</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Базу iTunes</string>
    </property>
   </item>
  </widget>
 </widget>
 <tabstops>
  <tabstop>comboBox</tabstop>
  <tabstop>iTunesMediaPathEdit</tabstop>
  <tabstop>iTunesMediaPathButton</tabstop>
  <tabstop>musiclibPathEdit</tabstop>
  <tabstop>musiclibPathButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
